osquery/tables/system/windows/startup_items.cpp

```diff
@@ -57,6 +58,26 @@ const std::set<std::string> kStartupStatusRegKeys = {
 // Starts with 0[0-9] but not followed by all 0s
 const auto kStartupDisabledRegex = boost::regex("^0[0-9](?!0+$).*$");
 
+static inline void parseStartupPath(const std::string& path, Row& r) {
+  if (path.find('\"') == std::string::npos) {
+    r["path"] = path;
+  } else {
+    boost::tokenizer<boost::escaped_list_separator<TCHAR>> tokens(
```

Comment:
The problem with split is you can only split on one delimeter. The issue I have here is I need to consider any delims (i.e. spaces) inside quotes as one token.

i.e.
"C:\Program Files\osquery\osqueryd\osqueryd.exe" --flagfile osquery.flags
The first token is "C:\Program Files\osquery\osqueryd\osqueryd.exe" , the rest are seperated by spaces as usual.

However not everything has double quotes either, so that case also needs ot be handled.

I found it cleaner to use tokenizer than splitting on optional '"' and escaping spaces, but maybe there is an easy way to do this that I'm missing?